### ISSUE TITLE
[Boss] The Crown from Opera Event in Karazhan

### ISSUE THREAD

**Issue by MoiInActie** [WoWManiaUK/Redemption#1986]
Whilst soloing Karazhan I arrived at Opera Event and got The Crown. The first 4 mobs are targettable, will attack me and I can attack them. But when they die, the Crown spawns and she is was attackable and she also didn't attack me. Due to this, the event couldn't be finished, the doors didn't open. I teleported out and ran back to the instance, but then it kept saying "encounter in progress".

Later I went inside with a GM and I got another event (Big Bad Wolf), which worked fine. Been able to then kill that boss and continue. So for now it seems that the issue is specifically with The Crown event from Opera.

**Comment by fvicaria** [comment]
Most of these bosses were not made to be solo'ed. If you over dps you will cause them to skip events and bug. 
What was the boss exactly send me links as requested.

Please follow the procedure described below before creating a new issue:
[How to Properly Report an Issue](https://github.com/wow-mania/Redemption/wiki/How-to-Properly-Report-an-Issue)

This will help developers identify and fix the issue quicker. Incorrectly or unclear reports will simply be deleted.

**Comment by MoiInActie** [comment]
The exact boss is this: http://www.wowhead.com/npc=18168/the-crone

I did not over dps it or anything like that, soloing as a protection paladin doing around 3k dps is lower then a full lvl 70 raidgroup would do. Killing the 5 mobs that spawned first took (I guess) somewhere around 3 minutes. After all of them die, the Crone spawns. She did spawn, but did nothing and wasn't killable. I believe she normally starts talking and casting some spells.

I don't want to exclude the option that it got bugged because I was soloing it, but I don't see a direct reason why it should. Bosses that do a mindcontrol often don't work properly when soloing, but to my knowledge this boss doesn't do anything like that.

To answer some more questions from the  How To:
- I did use the correct client/launcher
- I tried to replicate the issue with a GM in the raid, but got another event (Big Bad Wolf) so couldn't reproduce it

After the reset next Tuesday I can try and see if I can reproduce the issue again and make some screenshots if needed. This ofcourse IF I get the right event at Karazhans Opera Event, which is random.
